<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JKNewProject</class>
 <widget class="QDialog" name="JKNewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JKNewProject</string>
  </property>
  <widget class="QDoubleSpinBox" name="dSpBxStampTax">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string>印花税千分之</string>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dSpBxTransfer">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string>过户税千分之</string>
   </property>
   <property name="value">
    <double>0.020000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dSpBxCommission">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>61</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string>佣金税千分之</string>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dSpBxAlertPercent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string>预警百分比</string>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="value">
    <double>3.000000000000000</double>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>204</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pBtnOK">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBtnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
